.travis.yml

```diff
@@ -239,15 +240,19 @@ matrix:
   - name: "Java w/ Oracle JDK 9"
```

Comment:
Should we update the name to "OpenJDK 9"?

Reply:
I just fixed